Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/ReflectometryISISPreprocessTest.py

```diff
@@ -10,7 +10,7 @@
 from mantid.api import AnalysisDataService, IEventWorkspace, MatrixWorkspace, WorkspaceGroup
 from mantid.simpleapi import CreateSampleWorkspace
 from plugins.algorithms.WorkflowAlgorithms.ReflectometryISISPreprocess import ReflectometryISISPreprocess
-from testhelpers import create_algorithm
+from testhelpers import (assertRaisesNothing, create_algorithm)
```

Comment:
Nit, could we just do 
`alg.execute` without using `assertRaisesNothing`?

This is something I'm gradually trying to cull across the codebase and remove. PyTest will stop the test if any exception is thrown, handles the exception better than we do (since internally it just echos the method and coerces the exception to a string) and gives people an error message they can search online rather than our bespoke one